test/SignerBondsUniswapV2.test.js

```diff
@@ -74,6 +74,181 @@ describe("SignerBondsUniswapV2", () => {
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when slippage tolerance too big", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(governance)
+            .setSlippageTolerance(maxBasisPoints + 1)
+        ).to.be.revertedWith("Maximum value is 10000 basis points")
+      })
+    })
+
+    context("when governance did not set slippage tolerance", () => {
+      const defaultSlippageTolerance = 50
+      it("should return the default value", async () => {
+        expect(await signerBondsUniswapV2.slippageTolerance()).to.be.equal(
+          defaultSlippageTolerance
+        )
+      })
+    })
+
+    context("when slippage tolerance is in correct range", () => {
+      const priceImpact = 500
+      beforeEach(async () => {
+        expect(
+          await signerBondsUniswapV2
+            .connect(governance)
+            .setSlippageTolerance(priceImpact)
+        )
+      })
+
+      it("should set slippage tolerance", async () => {
+        expect(await signerBondsUniswapV2.slippageTolerance()).to.be.equal(
+          priceImpact
+        )
+      })
+    })
+  })
+
+  describe("setSwapDeadline", () => {
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2.connect(thirdParty).setSwapDeadline(10 * 60) // 10 min
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when governance did not set default swap deadline", () => {
+      const defaultSwapDeadline = 20 * 60 // 20 min
+      it("should return the default value", async () => {
+        expect(await signerBondsUniswapV2.swapDeadline()).to.be.equal(
+          defaultSwapDeadline
+        )
+      })
+    })
+
+    context("when governance set default swap deadline", () => {
+      const swapDeadline = 100 * 60 // 100 min
+      beforeEach(async () => {
+        expect(
+          await signerBondsUniswapV2
+            .connect(governance)
+            .setSwapDeadline(swapDeadline)
+        )
+      })
+
+      it("should set swap deadline", async () => {
+        expect(await signerBondsUniswapV2.swapDeadline()).to.be.equal(
+          swapDeadline
+        )
+      })
+    })
+  })
+
+  describe("setRevertIfAuctionOpen", () => {
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2.connect(thirdParty).setRevertIfAuctionOpen(false)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context(
+      "when governance did not set default value for revert if auction open",
+      () => {
+        const defaultRevertIfAuctionOpen = true
+        it("should return the default value", async () => {
+          expect(await signerBondsUniswapV2.revertIfAuctionOpen()).to.be.equal(
```

Comment:
It does describe `revertIfAuctionOpen`, not `setRevertIfAuctionOpen`.

Reply:
Fixed in https://github.com/keep-network/coverage-pools/pull/156/commits/5f635a73cfd5eebb12950e447acdec55b28b0cd5.